### ISSUE TITLE
Charge cell data with SQLite Android

### ISSUE THREAD

**Issue by bbf-developer** [evrencoskun/TableView#59]
How to code for communicating tableview with SQLite?

**Comment by bbf-developer** [comment]
Example:

the user save the parameter of cell for later charge in a table with SQLite to continue the work.

**Comment by evrencoskun** [comment]
Hi @bbf-developer 

As far as I can understand your request, you want to show your SQLite data on a TableView. Am I right?

If I am, there is no any direct connection by SQLite DB to TableView. Unfortunately, you should build your own.

**Comment by bbf-developer** [comment]
Yes, i develop the method directly into a App, only i need where starting, thanks a lot @evrencoskun

**Comment by sonique6784** [comment]
@bbf-developer , can we close this issue?

**Comment by getuget** [comment]
@bbf-developer have you succeed in communicating Sqlite data with TableView?

I have similar idea but do not know where to start
